lib/net/http.rb

```diff
@@ -497,22 +517,25 @@ def HTTP.post(url, data, header = nil)
       }
     end
 
-    # Posts HTML form data to the specified URI object.
-    # The form data must be provided as a Hash mapping from String to String.
-    # Example:
-    #
-    #   { "cmd" => "search", "q" => "ruby", "max" => "50" }
+    # Posts data to a host; returns a Net::HTTPResponse object.
     #
-    # This method also does Basic Authentication if and only if +url+.user exists.
-    # But userinfo for authentication is deprecated (RFC3986).
-    # So this feature will be removed.
+    # Argument +uri+ must be a URI;
```

Comment:
```suggestion
    # Argument +url+ must be a URI;
```

Reply:
Clearer to rename the argument via call-seq?

Reply:
I think this call seq doesn't include any additional info.

Reply:
Does it not correctly name the argument?  (Or am I missing something?)

Reply:
We can use the RDoc generated call-seq instead of the custom call-seq (since the custom call-seq doesn't provide additional info). The method definition names the first argument `url` and not `uri`.

Reply:
But aren't we also saying that the first argument (called url in the code) is and must be actually a URI?  So renaming omits a possibly confusing difference?

Reply:
In other words, I don't think we should have to say, "Argument +url+ must be a URI";  I think it's better to be able to say, "Argument +uri+ must be a URI."

Reply:
I agree that "Argument +uri+ must be a URI." sounds better. However, keeping the argument name as `url` is more consistent with the rest of the code base and more clear that we're accepting an URL and not any URI.

Reply:
Done.